modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,182 @@
+
+.Possible pod volume backup failure if Velero is installed in several namespaces
+
+There was a regresssion in Pod Volume Backup (PVB) functionality when Velero was installed in several namespaces. The PVB controller was not properly limiting itself to PVBs in its own namespace.
+link:https://issues.redhat.com/browse/OADP-2308[OADP-2308]
+
+.OADP Velero plugins returning "received EOF, stopping recv loop" message
+
+In OADP, Velero plugins were started as separate processes. When the Velero operation completes, either successfully or not, they exit. Therefore, if you see a `received EOF, stopping recv loop` messages in debug logs, it does not mean an error occurred, it means that a plugin operation has completed. link:https://issues.redhat.com/browse/OADP-2176[OADP-2176]
+
+.CVE-2023-39325 Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack)
+In previous releases of OADP, the HTTP/2 protocol was susceptible to a denial of service attack because request cancellation could reset multiple streams quickly. The server had to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection. This resulted in a denial of service due to server resource consumption
+
+The following issues have been listed under this issue:
+
+* link:https://issues.redhat.com/browse/OADP-2886[OADP-2886]
+* link:https://issues.redhat.com/browse/OADP-2895[OADP-2895]
+* link:https://issues.redhat.com/browse/OADP-2869[OADP-2869]
+* link:https://issues.redhat.com/browse/OADP-2898[OADP-2898]
+* link:https://issues.redhat.com/browse/OADP-2904[OADP-2904]
+* link:https://issues.redhat.com/browse/OADP-2907[OADP-2907]
+* link:https://issues.redhat.com/browse/OADP-2910[OADP-2910]
+* link:https://issues.redhat.com/browse/OADP-2889[OADP-2889]
+* link:https://issues.redhat.com/browse/OADP-2892[OADP-2892]
+* link:https://issues.redhat.com/browse/OADP-2901[OADP-2901]
+
+For more information, see link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325 (Rapid Reset Attack)]
+
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12422837[OADP 1.3.0 resolved issues] in Jira.
+
+[id="known-issues-1-3-0_{context}"]
+== Known issues
+
+.CSI plugin panics on nil pointer when csiSnapshotTimeout is set to a short duration
+The CSI plugin panics on nil pointer when `csiSnapshotTimeout` is set to a short duration. Sometimes it succeeds to complete the snapshot within a short duration, but often it panics with the backup `PartiallyFailed` with the following error: `plugin panicked: runtime error: invalid memory address or nil pointer dereference`. link:https://issues.redhat.com/browse/OADP-3069[OADP-3069]
```

Comment:
I am not sure if we can use the word `panic`. Can we use the word `error` instead?

Reply:
It would be better to use "error."